recipes/ntp/ntp.inc

```diff
@@ -45,6 +45,14 @@ do_install_extra_files() {
     install -m 755 ${SRCDIR}/ntpd ${D}/${sysconfdir}/init.d/ntpd
 }
 
+RECIPE_FLAGS +="ntpd_start_args"
```

Comment:
While I do see that there is some amount of redundancy in it, I would like to suggest using the name USE_ntp_ntpd_start_args instead of USE_ntpd_start_args.  The reason for this, is that there are other ntpd implementations out there (fx. the one in busybox), and they are not fully argument compatible, so setting ntpd_start_args in fx. a DISTRO could be problematic.